worth2/main/views.py

```diff
@@ -3,14 +3,22 @@
 from django.core.urlresolvers import reverse
 from django.views.generic.base import TemplateView
 from django.views.generic.list import ListView
-from django.shortcuts import get_object_or_404, redirect
+from django.shortcuts import get_object_or_404, redirect, render
 
 from pagetree.generic.views import PageView
+import quizblock.models as qm
```

Comment:
Sorry, missed this last night. Others can weigh in on this - but I'm strongly opposed to importing with obscure names. "qm" reads as a local variable in the code, not a set of globally imported models.